pandas/util/_print_versions.py

```diff
@@ -114,14 +114,13 @@ def show_versions(as_json=False):
 
     else:
         maxlen = max(len(x) for x in deps)
-        tpl = "{{k:<{maxlen}}}: {{stat}}".format(maxlen=maxlen)
         print("\nINSTALLED VERSIONS")
         print("------------------")
         for k, stat in sys_info:
-            print(tpl.format(k=k, stat=stat))
+            print(f"{k}:<{maxlen}: {stat}")
```

Comment:
Does this print the same as previous? Previous output has extra {}?

Reply:
Hi
I looked into it changed it.
This is my first time contributing so thanks for the feedback.